### ISSUE TITLE
Radio buttons render as a text input in IE8

### ISSUE THREAD

**Issue by jerel** [emberjs/ember.js#12295]
Here is a very tiny example: http://emberjs.jsbin.com/kiyesi 

`<input type="radio">` of course renders fine in IE8 but rendering the same minimal example dynamically via a component results in a text input instead:

``` js
App.RadioCheckComponent = Ember.Component.extend({
  tagName: 'input',
  type: 'radio',
  attributeBindings: ['type'],
});
```

I just upgraded from Ember v1.12.x and the problem did not exist there.

**Comment by pixelhandler** [comment]
Is there a quick way for someone to see this emulated in IE8 (don't personally have a PC) ?

**Comment by jherdman** [comment]
https://github.com/xdissent/ievms ought to help you out.

**Comment by pixelhandler** [comment]
@jherdman I was curious if there was anything quicker (than installing) perhaps web based. Really don't want to install IE8 :(

**Comment by jerel** [comment]
@pixelhandler here is a screenshot of the result and the code: 
![ie8](https://cloud.githubusercontent.com/assets/322706/9693349/4c77dbf2-5315-11e5-9221-c29e60ef5cc0.png)

As far as I can tell the resulting html is correct. I've compared between my v1.12.x and v1.13.x versions. I'm guessing it has something to do how the element is created (timing of when the type is added perhaps) but that's just a guess.

**Comment by eddflrs** [comment]
@pixelhandler There's the free trial browserstack offers: https://www.browserstack.com/

**Comment by rwjblue** [comment]
This is likely the same as https://github.com/tildeio/htmlbars/issues/380 which came from https://github.com/emberjs/ember.js/issues/11553.

**Comment by pixelhandler** [comment]
@eddflrs yeah would be cool if could group fund browserstack for ember debugging as a community

**Comment by pixelhandler** [comment]
@jerel yeah `<input id="ember304" type="radio" class="ember-view">` is not the same as `<input id="ember304" type="radio" class="ember-view"></input>` but other browsers do display that correctly either way.

**Comment by jerel** [comment]
This does appear to be caused by Ember trying to set `type` after the element is in the DOM.

As a temporary workaround I implemented a component with `tagName: ''` and put `<input type="radio" dynamic="{{foo}}">` in its template and that works.

**Comment by gauthierm** [comment]
This should be fixed in the latest https://github.com/emberjs/ember.js/pull/12351

**Comment by rwjblue** [comment]
Fixed by #12596.
